Documentation/prereqs.md

```diff
@@ -0,0 +1,51 @@
+.NET Core native prerequisites
+==============================
+
+This document outlines the dependencies needed to run .NET Core tools and applications. 
+
+## Windows dependencies
```

Comment:
Where are we calling out the list of supported OS platforms and architectures since that is part of the pre-req as well?